src/styles/animations.js

```diff
@@ -0,0 +1,69 @@
+var AutoPrefix = require('./auto-prefix');
+
+var createdAnimations = {};
+
+function createSheet(){
+
+	var styleElement = document.createElement("style");
+	document.head.appendChild(styleElement);
+	var sheet = styleElement.sheet;
+
+	return sheet;
+}
+
+function addRule(sheet, selector){
+	var size = (sheet.cssRules || sheet.rules).length;
+	if("insertRule" in sheet) {
+		sheet.insertRule(selector + " { }", size);
+	}
+	else if("appendRule" in sheet){
+		sheet.appendRule(selector + " { }");
+	}
+	else if("addRule" in sheet) {
+		sheet.addRule(selector, " {} ", size);
+	}
+}
+ 
+function setStyles(rule, obj){
+	for(var prop in obj){
+		rule.style[prop] = obj[prop];
+	}
+}
+
+module.exports = {
+
+	 create: function (name, frames){
+
+	 	if(name in createdAnimations){
+	 		return name;
+	 	}
+	 	
+	 	createdAnimations[name] = true;
+
+		var sheet = createSheet();
+		 
+		var keyframeTags = [ "@keyframes ", "@-webkit-keyframes "];
```

Comment:
@hai-cea @jkruder  I thought of doing so, but the performance loss worried me, in my test was very slow with setTimeout when had several elements on the screen. 
Do you guys believe that the implementation of a handler for animations is not viable?